src/services/contracts.ts

```diff
@@ -4,60 +4,57 @@ import {
   ApeDistributor__factory,
   ApeRouter,
   ApeRouter__factory,
-  ApeToken,
-  ApeToken__factory,
   ApeVaultFactoryBeacon,
   ApeVaultFactoryBeacon__factory,
   ApeVaultWrapperImplementation,
   ApeVaultWrapperImplementation__factory,
   ERC20,
   ERC20__factory,
 } from '@coordinape/hardhat/dist/typechain';
+import debug from 'debug';
 import * as ethers from 'ethers';
 
-import { getToken, NetworkId } from 'config/networks';
+import { HARDHAT_CHAIN_ID, HARDHAT_GANACHE_CHAIN_ID } from 'config/env';
+
+const log = debug('coordinape:contracts');
 
 type SignerOrProvider = ethers.providers.Provider | ethers.ethers.Signer;
 
+export const supportedChainIds: number[] =
+  Object.keys(deploymentInfo).map(Number);
+
 export class Contracts {
-  usdc: ERC20;
-  apeToken: ApeToken;
-  apeVaultFactory: ApeVaultFactoryBeacon;
-  apeRouter: ApeRouter;
-  apeDistributor: ApeDistributor;
+  vaultFactory: ApeVaultFactoryBeacon;
+  router: ApeRouter;
+  distributor: ApeDistributor;
 
-  // TODO this might not be quite the right way to do this, as the signer/provider
-  // used to create the contracts also has a network associated with it
-  networkId: NetworkId;
+  // TODO this might not be quite the right way to do this, as the
+  // signer/provider used to create the contracts also has a network associated
+  // with it
+  chainId: number;
 
   signerOrProvider: SignerOrProvider;
 
   constructor(
     contracts: {
-      usdc: ERC20;
```

Comment:
this was just given its own prop for convenience while this file was being developed--now we can use `getToken` below instead

Reply:
`apeToken` below was removed entirely because we don't have a need for it yet